xml/api/fence-event-2.28.rng

```diff
@@ -30,6 +30,11 @@
             <attribute name="origin"> <text /> </attribute>
             <optional>
                 <attribute name="completed"> <text /> </attribute>
+                <optional>
+                    <attribute name="completed-ns">
+                        <data type="nonNegativeInteger" />
```

Comment:
Is this okay or should we open this up to `integer`? It comes from `stonith_history_t:completed_nsec`, which is `long`. It should never be negative (in fact, text output should assert via `crm_time_as_string()` if it is), but we've been burned before.

Reply:
Since completed is a string I'm fine with just updating that. If we separated out all the time components (year/month/etc.), it would make sense to add completed-ns (in retrospect, it would be more XMLic to separate all of them, but I wouldn't bother now unless someone asks for it).

Reply:
That's preferable, I was just afraid to mess with the time format in public API. Sounds good.